### DISCUSSION TITLE
Reset when using external React Select dropdown

### DISCUSSION THREAD

**Discussion by keerthankrishna** [react-querybuilder/react-querybuilder#497] [General]
The reset props ` resetOnFieldChange ` and ` resetOnOperatorChange ` work well when using the default ` QueryBuilder ` input but don't work when I use a _react_ ` Select ` component which is an external component. Any direction or suggestion on how i could get the reset done when using an external _React_ ` Select ` component ?

**Comment by jakeboone02** [comment]
Please provide a working example. You can start with one of the [examples here](https://github.com/react-querybuilder/react-querybuilder/tree/main/examples).
